README.md

```diff
@@ -219,10 +245,13 @@ SET error_on_unknown_object_key=false;
 SELECT * FROM "ext"."retention_policy" WHERE tags['foo'] IS NOT NULL;
 ```
 
-Delete all retention policy records tagged with `foo`.
+Delete all retention policy records tagged with `foo` and `bar`.
 ```sql
 DELETE FROM "ext"."retention_policy" WHERE tags['foo'] IS NOT NULL;
```

Comment:
Should be `WHERE tags['foo'] IS NOT NULL AND tags['bar'] IS NOT NULL`?

Reply:
In this case, the SQL statement is `DELETE FROM "ext"."retention_policy" WHERE tags['foo'] IS NOT NULL;`, so it intends to only delete records tagged with `foo`. That it is essentially the same record which is also tagged with `bar` is just a coincidence.

Reply:
Ah, you are suggesting to amend the SQL statement. Now, I am getting it. Yeah, sure, why not.

Reply:
Fixed. Thank you.